### DISCUSSION TITLE
京都大學正在開發「能理解人類幽默感細微差異」的AI系統，訓練機器人在正確時機發笑

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#699] [General]
一般的機器人或AI系統，通常能與我們對答如流，然而卻是冷冰冰的，我們是感受不到他的情的。
這項機器人研究的領導人說：「對話式人工智慧中最重要的功能即是同理心，因此認定機器人能與用戶共同歡笑是展現同理的指標，而這是文本訊息的聊天機器人無法做到的事。」
這個研究中，主要是用：共同發笑模型（shared-laughter model），這個模型有三個要點：1.機器人需能辨認笑聲 2.能決定是否要笑或沉默 3.能選擇合適的笑法。如此一來，當人類發出笑聲的同時，人工智慧能給予適當且具同理心的回應。

之前有看到同學有發可以陪伴小孩的機器人，我覺得加上同理心，會笑的機器人，應該會更適合陪伴小朋友。

文章連結：https://www.thenewslens.com/article/173746

**Comment by titan92601** [comment]
這位同學，您的這個貼文看似是紅色的，但是不論是好的貼文或是不那麼好的貼文，都是能夠促進我們這個社群，真的要感謝老師給了我們這個機會，才能在這討論，也才譨在這找到知己。

**Comment by Vincent20030314** [comment]
讓機器人可以和人用表情互動確實不錯，不過如果表情做得不夠好，可能會導致使用者反感。
